docker-for-windows/index.md

```diff
@@ -25,6 +25,16 @@ Please read through these topics on how to get started. To **give us your feedba
 
 >**Already have Docker for Windows?** If you already have Docker for Windows installed, and are ready to get started, skip over to the [Getting Started with Docker](/engine/getstarted/index.md) tutorial.
 
+## Using pre-installed Docker on Windows Server 2016 or Windows 10
+
+Windows Server 2016 and Windows 10 include a native Docker daemon, which allows
```

Comment:
@mstanleyjones I don't thinkg "include" is the best work - they still have to be installed. Maybe "Windows Server 2016 and Windows 10 can run the Docker daemon natively"